addons/advanced_throwing/script_component.hpp

```diff
@@ -23,11 +23,15 @@
 #define THROWSTYLE_HIGH_DIR [0, 200, 500]
 #define THROWSTYLE_HIGH_VEL_COEF 2
 #define THROWSTYLE_DROP_VEL 2
-#define THROWSTYLE_HIGH_TORQUE_COEF .6
+#define THROWSTYLE_DROP_STEP 0.1
```

Comment:
This is just an possibility: could make it a setting
probably don't need the snap/new hints then as someone can just set it to 1 to keep old behavior

```suggestion
#define THROWSTYLE_DROP_STEP (0.8/GVAR(throwStepSetting))
```
```
[
    QGVAR(throwStepSetting), "SLIDER",
    ["throw steps", "..."],
    _category,
    [1, 10, 8, -1, false],
    0
] call CBA_fnc_addSetting;
```